src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupService.java

```diff
@@ -1297,4 +1297,183 @@ public static ApiElastigroup importEC2Instance(ElastigroupImportEC2Instance requ
         return retVal;
 
     }
+
+    public static Boolean pauseStatefulInstance(String groupId, String statefulInstanceId,
+                                         String authToken, String account) {
+
+        Boolean retVal = false;
+
+        // Get endpoint
+        SpotinstHttpConfig config      = SpotinstHttpContext.getInstance().getConfiguration();
+        String             apiEndpoint = config.getEndpoint();
+
+        // Build query params
+        Map<String, String> queryParams = new HashMap<>();
+
+        // Add account Id Query param
+        if (account != null) {
+            queryParams.put("accountId", account);
+        }
+
+        // Get the headers
+        Map<String, String> headers = buildHeaders(authToken);
+
+        // Build URI
+        String uri = String.format("%s/aws/ec2/group/%s/statefulInstance/%s/pause", apiEndpoint, groupId, statefulInstanceId);
+
+        // Send the request.
+        RestResponse response = RestClient.sendPut(uri, "", headers, queryParams);
```

Comment:
RestResponse to be casted to BaseServiceEmptyResponse before parsing to get status

BaseServiceEmptyResponse emptyResponse = getCastedResponse(response, BaseServiceEmptyResponse.class);